cmd/otel-agent/subcommands/run/command.go

```diff
@@ -92,7 +92,23 @@ func (o *orchestratorinterfaceimpl) Reset() {
 }
 
 func runOTelAgentCommand(ctx context.Context, params *subcommands.GlobalParams, opts ...fx.Option) error {
-	err := fxutil.Run(
+	acfg, err := agentConfig.NewConfigComponent(context.Background(), params.CoreConfPath, params.ConfPaths)
+	if err == agentConfig.ErrNoDDExporter {
+		return fxutil.Run(
+			fx.Provide(func() []string {
+				return append(params.ConfPaths, params.Sets...)
+			}),
+			collectorcontribFx.Module(),
+			collectorfx.ModuleNoAgent(),
+			fx.Options(opts...),
+			fx.Invoke(func(_ collectordef.Component) {
+			}),
+		)
+	} else if err != nil {
```

Comment:
nit: may be do 
```
if err != nil && err != agentConfig.ErrNoDDExporter{
		return err
	}

if err == agentConfig.ErrNoDDExporter {
...
}


```
Not a fan of else blocks

Reply:
done